matscholar_web/search/logic.py

```diff
@@ -115,3 +118,16 @@ def search_bar_live_display(example_search_n_clicks, *ent_txts):
         return entry
     else:
         return random.choice(example_searches)
+
+
+def is_cobalt_search(entity_query, raw_text):
+    if raw_text is not None:
+        if 'Co' in raw_text.split() or 'cobalt' in raw_text.lower().split():
+            return True
+
+    if entity_query is not None:
+        if 'material' in entity_query.keys():
+            if any(['Co' in m for m in entity_query['material']]) or any(['cobalt' in m.lower() for m in entity_query['material']]):
```

Comment:
```
any(['Co' in m for m in entity_query['material']])..
```

Won't this throw a cobalt warning for things like "Colloidal", if it's entered as a material?

This line might work better as a more precise regex or something
@AmalieT

Reply:
Good point. Changed to use a regex as suggested.